src/main/java/io/github/moulberry/notenoughupdates/mixins/MixinThreadDownloadImageDataThread.java

```diff
@@ -37,7 +37,8 @@ public class MixinThreadDownloadImageDataThread {
 			value = "INVOKE",
 			target = "Ljava/net/HttpURLConnection;setDoOutput(Z)V"
 		),
-		locals = LocalCapture.CAPTURE_FAILSOFT
+		locals = LocalCapture.CAPTURE_FAILSOFT,
+		remap = false
```

Comment:
It is under ``net.minecraft`` tho, pretty sure most if not all class files under that package are actually mapped.
If it was intended for the invoked target it should've been added in the ``@At`` interface and not outside.
But it does not seem to actually change anything.

Reply:
This is a java function that is being overridden, and removing this breaks compiling.